src/bundler/deno_bundler_test.ts

```diff
@@ -0,0 +1,71 @@
+import { assertRejects, assertSpyCalls, stub } from "../dev_deps.ts";
+import { BundleError } from "../errors.ts";
+import { DenoBundler } from "./deno_bundler.ts";
+
+Deno.test("Deno Bundler tests", async (t) => {
+  await t.step(DenoBundler.bundle.name, async (tt) => {
+    await tt.step(
+      "should invoke 'deno bundle' successfully",
```

Comment:
Technically we are not testing whether `bundle` is invoked here, we are testing whether `Deno.Command` was invoked. I think we may want to assert on the arguments passed into the `Command` stub to make sure it is `bundle` - assuming that's the right hypothesis we are testing.

Reply:
Makes sense, Ive added argument validation 💯